### PR TITLE
Data provider detail layout

### PR DESCRIPTION
**Pull request by Joozty**
This PR contains:
1. layout for data providers details page
2. Load first 10 outputs
3. search box (without functionality)
4. map

Missing parts (part of following PRs):
1. pagination
2. search working
3. ...

@viktor-yakubiv I decided to leave map interactive because I find it quite useful but if you insist I can make it non-interactive. Let me know. Also, it's quite a big PR and everything is in one commit. Sorry about that. I was doing too many changes and lost track of it. I don't expect you to go through the code rather concentrate more on look.

### CODE REVIEW

**Comment by viktor-yakubiv** [comment]
> I decided to leave map interactive because I find it quite useful but if you insist I can make it non-interactive.

☝️ This is good. I agree that the interactive map is better.

> Also, it's quite a big PR and everything is in one commit. Sorry about that. I was doing too many changes and lost track of it. 

☝️ This is fine as well, it happens. Sometimes you make changes that touch a lot of stuff, sometimes you do a big feature like this one.

---

Could you please remove the search field and pagination so this PR looks like finished work and bring it back in the following PRs?

**Comment by Joozty** [comment]
>1. Could you please remove the search field and pagination so this PR looks like finished work and bring it back in the following PRs?

I removed pagination but left search because I think it's fine. It's part of the layout and this PR is going to be merged to a different branch than master so it's fine to have some work in progress there.

> 2. Could you please describe your work in the commit message more precisely? What components are created? How are they used? What libraries were added and so on?

I am not much a wordy guy. I understand your point but on the other hand, I don't think this is something which is stopping us at the moment. It's up to everyone how they handle commits/messages. 🙂

File: templates/data-provider/claim-card.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './styles.module.css'
+
+const ClaimCard = ({ name }) => (
+  <Card className={classNames.use(styles.card, styles.claimCard)}>
+    <div>
+      <b>Are you {name}&apos;s manager?</b> Access insider analytics, issue
+      reports and manage access to documents from your repository in the CORE
+      Dashboard! 👇
```
Comment (viktor-yakubiv) [inline templates/data-provider/claim-card.jsx:12]: This text has been changed:

```suggestion
      <b>Do you manage {name}?</b> Access insider analytics, issue
      reports and manage access to outputs from your repository in
      the&nbsp;CORE&nbsp;Dashboard!&nbsp;
      <span aria-hidden="true" role="img">👇</span>
```

Also, I added [accessibility to the emoji](https://www.npmjs.com/package/a11y-react-emoji).

File: templates/data-provider/claim-card.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Card, Button } from '@oacore/design'
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './styles.module.css'
+
+const ClaimCard = ({ name }) => (
+  <Card className={classNames.use(styles.card, styles.claimCard)}>
```
Comment (viktor-yakubiv) [inline templates/data-provider/claim-card.jsx:8]: If you use a card, it should be either a `<section>` or an `<aside>` section (I am not sure which).

However, it's designed in a way that this is a footer to `<aside>` which already contains sections and it should not have a spacing arount.

File: templates/data-providers/result-card.jsx [hunk 3]
```diff
@@ -17,19 +18,19 @@ const ResultCard = ({
...
       >
-        {title}
+        <UILink className={styles.resultTitle}>{title}</UILink>
       </Link>
     </span>
 
     <ul className={styles.resultMetadata}>
       <li>
-        <Link href={homePage} external icon={false}>
+        <UILink href={homePage} external icon={false}>
```
Comment (viktor-yakubiv) [inline templates/data-providers/result-card.jsx:31]: What is this?

**Review by viktor-yakubiv — APPROVED** [review]
It looks good to me apart from the comments in #118

File: templates/data-providers/result-card.jsx (hunk 3 above)
Comment (Joozty) [reply templates/data-providers/result-card.jsx:31]: This is link to the homepage of repository displayed data providers search page.

File: templates/data-provider/claim-card.jsx (hunk 2 above)
Comment (Joozty) [reply templates/data-provider/claim-card.jsx:8]: I was expecting that these sidebar cards can be positioned on different places on mobile view, i.e. claim card after two outputs results and so on hence I didn't consider it as a footer. 


I would say the whole sidebar should be aside and the inner card can stay `div`. 🤔

Comment (viktor-yakubiv) [reply templates/data-provider/claim-card.jsx:8]: According to the [spec](https://html.spec.whatwg.org/multipage/sections.html#the-aside-element), aside can contain nested sections (check what "[Flow content](https://html.spec.whatwg.org/multipage/dom.html#flow-content-2)" means).

Comment (Joozty) [reply templates/data-provider/claim-card.jsx:8]: ok why not thanks :)

**Pull request merged**